values.yaml.mako

```diff
@@ -0,0 +1,10 @@
+# A helm values file to be used with this chart:
+# https://github.com/camptocamp/private-geo-charts/tree/master/c2cgeoportal
+tilecloudchain:
+  s3:
+    enabled: true
```

Comment:
Since I don't have default credentials to put, yes, I've disabled it by default:
https://github.com/camptocamp/private-geo-charts/blob/master/c2cgeoportal/values.yaml#L59

Reply:
OK, thanks :-)